apps/admin-gui/src/app/shared/components/dialogs/add-role-dialog/add-role-dialog.component.ts

```diff
@@ -37,9 +38,12 @@ export class AddRoleDialogComponent implements OnInit {
 
   selectedRule: RoleManagementRules;
   selectedEntity: PerunBean = null;
+  selected = new SelectionModel<PerunBean>(true, []);
```

Comment:
The property `selectedEntity` is NOT necessary anymore. If you agree, you can delete it also from the HTML file.
```suggestion
  selected = new SelectionModel<PerunBean>(true, []);
```